### DISCUSSION TITLE
3번째 과제 그래프 관련 라이브러리(Chart.js) 입니다.

### DISCUSSION THREAD

**Discussion by hyoniiii** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#24] [Reference]
https://velog.io/@jinu_820/react%EC%97%90%EC%84%9C-chart.js-%EC%82%AC%EC%9A%A9%ED%95%B4%EB%B3%B4%EA%B8%B0

또 다른 라이브러리 있으면 공유 부탁드려요~!

**Comment by hyoniiii** [comment]
https://react-chartjs-2.js.org/

**Reply by hyoniiii** [reply]
https://velog.io/@eunjin/React-%EB%A6%AC%EC%95%A1%ED%8A%B8-%EA%B7%B8%EB%9E%98%ED%94%84%EC%B0%A8%ED%8A%B8-%EB%9D%BC%EC%9D%B4%EB%B8%8C%EB%9F%AC%EB%A6%AC-%EB%AA%A8%EC%9D%8C

검색하니까 되게 여러개 나오는데 다들 뭘로 작업중이신가요..?

**Comment by jaeyeong815** [comment]
아직 레이아웃 잡고있어요 ,, ㅎ 사이드바랑 메인 영역 먼저 나누고 하려구요 ,,

**Comment by baechanghyeon** [comment]
저도 아직 레이아웃 잡고 있습니다..ㅎㅎ
